### PR TITLE
Update ms

### PR DESCRIPTION
**Pull request by palango**
This adds the requested overview to the MS spec.

I also tried to remove old information and make it more readable in general. Please let me know if I managed to do that.

### CODE REVIEW

File: monitoring_service.rst [hunk 1]
```diff
@@ -4,36 +4,24 @@ Monitoring Service
...
-    is expected to react. This way the settleChannel function that calculates the token distribution can only be
-    executed submitting the corresponding balance values of the hashed balances provided before
-5)  Either Alice or Bob (or anyone else) can now anytime call ``settleChannel`` and initiate the token distribution
-6)  After a successful submission of the hashed balance by the MS calling ``updateNonClosingBalanceProof`` (might be a
-    defeat of an attack), the monitoring service gets an on-chain payment from a smart contract where Alice deposited
-    some tokens beforehand
-7)  This will result in a correct distribution of tokens, in accordance with the off-chain transfers
+Whenever a channel is closed by calling ``closeChannel`` and if the client did
+not react itself, the MS will call ``updateNonClosingBalanceProof`` with the
+submitted MR on behalf of its client. For that action then MS can then claim the
```
Comment (pirapira) [inline monitoring_service.rst]: I see one too many `then`.  Maybe drop the first one?

File: monitoring_service.rst [hunk 2]
```diff
@@ -42,157 +30,62 @@ Information Flow
...
-
-When multiple MSs try to settle the same dispute, only the first one doing so receives a reward, but all of them have to
-invest resources to monitor the channel and spend gas to interact with the SC. If you find a way to continuously front
-run other MSs, you can drain their resources and block their only income. However, while doing so you fulfilled the MS’s
-duty to settle the payment channel correctly and protect the client’s deposit.
-In the short run, this is an acceptable outcome for the client. In the long run, this will drive other MSs out of
-business and thus reduce redundancy and reliability of the overall MS ecosystem. Since all MSs try to be the first to
-submit a BP, it is unlikely that a single MS will continuously be the fastest, but slightly slower MSs will still not
-get any rewards even if they are well behaved and reliable.
+The MS can claim its reward after successfully submitting its client’s balance proof update. This is only allowed when the Monitoring Service is registered in the Service Registry. For more infos see the :ref:`ServiceRegistry` contract.
```
Comment (pirapira) [inline monitoring_service.rst]: `claim()` works even after the MS's registration expires.

`monitor()` requires a valid registration.

File: monitoring_service.rst [hunk 3]
```diff
@@ -42,157 +30,62 @@ Information Flow
...
+Message Format
+==============
 
-Event Filtering
-'''''''''''''''
-MS must filter events for each on-chain channel that corresponds to the submitted balance proofs.
-On ``ChannelClosed`` and ``NonClosingBalanceProofUpdated`` events state the channel was closed with the Monitoring
-Service must call ``updateNonClosingBalanceProof`` with the respective latest balance proof provided by its client.
-On ``ChannelSettled`` event any state data for this channel MAY be deleted from the MS.
+Monitoring Services uses JSON format to exchange the data. For description of
```
Comment (pirapira) [inline monitoring_service.rst]: Either "Services-->Service" or "uses-->use".

File: monitoring_service.rst [hunk 4]
```diff
@@ -4,36 +4,24 @@ Monitoring Service
...
-    additional features and can be developed independently after Ithaca.
-
-Usual Scenario
-==============
+Overview
+========
+
+Monitoring services watch open payment channels when the user is not online. In
+case one channel partner closes a channel while the counterparty is offline (or
+doesn’t react for 80% of the settlement timeout after close has been called),
```
Comment (karlb) [inline monitoring_service.rst]: The 80% is not the whole truth. We have a time range when it reacts and use an approximation for the settlement period. We could at least add a `~` to not trick anyone into thinking this is precise.

File: monitoring_service.rst [hunk 5]
```diff
@@ -4,36 +4,24 @@ Monitoring Service
...
+the monitoring service sends the latest balance proof to the channel smart
```
Comment (karlb) [inline monitoring_service.rst]:
```suggestion
the monitoring service sends the latest balance proof to the TokenNetwork
```
This is just as short and more precise.

File: monitoring_service.rst [hunk 6]
```diff
@@ -42,157 +30,62 @@ Information Flow
...
-submit a BP, it is unlikely that a single MS will continuously be the fastest, but slightly slower MSs will still not
-get any rewards even if they are well behaved and reliable.
+The MS can claim its reward after successfully submitting its client’s balance proof update. This is only allowed when the Monitoring Service is registered in the Service Registry. For more infos see the :ref:`ServiceRegistry` contract.
 
-If a client wants to waste the resources of MSs, he can first broadcast a BP with a high reward and keep more recent BPs
-to himself. When a dispute happens, he can wait for the MSs to act before submitting his latest BPs, which prevents the
-MSs from receiving a reward. Doing this at a large scale is expensive, since the client needs to open and close a
-payment channel for this at his own cost.
+The payment is paid out from a deposit in the :ref:`UserDeposit` Contract (UDC).
+Ideally, only one MS submits the latest BP to the SC to avoid unnecessary gas
```
Comment (karlb) [inline monitoring_service.rst]: Should be one of
* avoid unnecessary gas costs
* avoid wasting gas

File: monitoring_service.rst [hunk 7]
```diff
@@ -42,157 +30,62 @@ Information Flow
...
+usage (for more infos see the description of the :ref:`Monitoring Service contract <MonitoringServiceContract>`.
```
Comment (karlb) [inline monitoring_service.rst]:
```suggestion
usage (for more information see the description of the :ref:`Monitoring Service contract <MonitoringServiceContract>`.
```

File: monitoring_service.rst [hunk 8]
```diff
@@ -241,7 +134,7 @@ Reward Proof
 
 ::
 
-    ecdsa_recoverable(privkey, sha3_keccak("\x19Ethereum Signed Message:\n181"
+    ecdsa_recoverable(privkey, sha3_keccak("\x19Ethereum Signed Message:\n221"
```
Comment (karlb) [inline monitoring_service.rst]: Nice catch! These are so easy to miss when updating the spec.

File: monitoring_service.rst [hunk 9]
```diff
@@ -274,3 +167,88 @@ Fields
...
+
+Monitoring Service are required to have a synced Ethereum node with an enabled JSON-RPC interface. All blockchain
+operations are performed using this connection.
+
+Event Filtering
+'''''''''''''''
+
+MS must filter events for each on-chain channel that corresponds to the submitted balance proofs.
+On ``ChannelClosed`` and ``NonClosingBalanceProofUpdated`` events state the channel was closed with the Monitoring
+Service must call ``updateNonClosingBalanceProof`` with the respective latest balance proof provided by its client.
```
Comment (pirapira) [inline monitoring_service.rst]: Maybe they should call `MonitoringService.monitor()`.  If they directly call `TokenNetwork.updateNonClosingBalanceProof()` directly, they cannot get the reward anymore.

File: monitoring_service.rst [hunk 10]
```diff
@@ -274,3 +167,88 @@ Fields
...
+recognize if there is no such possible reward.
+The MS on the other hand can freely choose to ignore requests when the reward is too low, so he will only choose
+requests that he deems fairly rewarded. If the MS ignores the client’s request, the client keeps his deposit and it can
+be used by other MSs or for later BPs. In summary, the exchange is fair for both parties, but there is a high likelihood
+that no exchange will happen at all.
+
+Non-frameability
+----------------
+
+MSs can put the clients channel deposit at risk by ignoring all client requests. But since a MS can’t force other MSs to
```
Comment (pirapira) [inline monitoring_service.rst]: "client's"

File: monitoring_service.rst (hunk 1 above)
Comment (palango) [reply monitoring_service.rst]: Thanks. This should be an `the`.

File: monitoring_service.rst (hunk 9 above)
Comment (palango) [reply monitoring_service.rst]: Updates this above and removed this whole paragraph.

**Pull request merged**